### PR TITLE
add rustc version when printing tikv-server version

### PR DESCRIPTION
**Pull request by tennix**
This makes `tikv-server -V` print following message, thus let users know which rustc builds tikv-server binary
```
Git Commit Hash: bf2e5e8e58ce2649385d3ad592ea22652fa1b9ef
UTC Build Time:  2017-01-23 03:46:47
Rustc Version:   1.15.0-nightly (71c06a56a 2016-12-18)
```

### CODE REVIEW

**Comment by tennix** [comment]
PTAL @overvenus @siddontang

**Comment by siddontang** [comment]
LGTM

**Comment by siddontang** [comment]
PTAL @BusyJay

File: build.rs [hunk 1]
```diff
@@ -44,3 +47,13 @@ fn commit_hash() -> String {
     cmd.args(&["rev-parse", "HEAD"]);
     cmd.output().ok().and_then(|o| String::from_utf8(o.stdout).ok()).unwrap_or("None".to_owned())
 }
+
+fn rustc_version() -> String {
+    let mut cmd = Command::new("rustc");
```
Comment (BusyJay) [inline build.rs]: Use environment `RUSTC` instead.

**Comment by siddontang** [comment]
PTAL @BusyJay

**Comment by BusyJay** [comment]
LGTM

**Pull request merged**